tests/test_main.py

```diff
@@ -12,15 +11,15 @@
 
 
 @pytest.fixture(autouse=True)
-def mock_settings_env_vars() -> Generator[None]:
-    mock_env = {
-        "POSTGRES_USER": "test_user",
-        "POSTGRES_PASSWORD": "test_password",
-        "POSTGRES_DB": "test_db",
-        "POSTGRES_PORT": "5432",
-        "POSTGRES_SERVER": "localhost",
-    }
-    with mock.patch.dict(os.environ, mock_env):
+def mock_db_config():
+    with (
+        mock.patch("src.service.POSTGRES_USER", "test_user"),
+        mock.patch("src.service.POSTGRES_PASSWORD", "test_password"),
+        mock.patch("src.service.POSTGRES_DB", "test_db"),
+        mock.patch("src.service.POSTGRES_PORT", "5432"),
+        mock.patch("src.service.POSTGRES_SERVER", "localhost"),
+        mock.patch("src.service.DATABASE_URL", "postgresql://test_user:test_password@localhost:5432/test_db"),
```

Comment:
utilize this
https://docs.python.org/3/library/unittest.mock.html#patch-multiple